components/StartPage/Header/index.js

```diff
@@ -0,0 +1,5 @@
+import Header from "../../QuizCardPage/Header";
+
+export default function HeaderStartPage() {
+  return <Header />;
+}
```

Comment:
I don't see any purpose in this: you can also import the `Header` on the page where you want to display it; there is no need for a wrapper component which does not do anything.